frontend/src/libs/components/profile-settings-sidebar/profile-settings-button/profile-settings-button.tsx

```diff
@@ -0,0 +1,40 @@
+import { useCallback } from 'react';
+
+import { Icon } from '#libs/components/components.js';
+import { type IconNames } from '#libs/enums/enums.js';
+
+import styles from './styles.module.scss';
+
+type ProfileSettingsButtonProperties = {
+  children: React.ReactNode;
+  name: IconNames;
+  onClick: (name: IconNames) => void;
+  isActive?: boolean;
+};
+
+const ProfileSettingsButton: React.FC<ProfileSettingsButtonProperties> = ({
+  name,
+  children,
+  onClick,
+  isActive = false,
+}) => {
+  const handleOnClick = useCallback(() => {
+    onClick(name);
+  }, [name, onClick]);
+
+  const buttonStyle = isActive
+    ? `${styles['profile-button']} ${styles['focused']}`
+    : styles['profile-button'];
+
+  return (
+    <button className={buttonStyle} onClick={handleOnClick}>
+      <Icon name={name} className={'profile-button-background'} />
```

Comment:
How is it working properly?

Reply:
Not sure how but it does 

https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/62558753/8e609586-15d5-4c28-9d26-62d663b68ee7

Reply:
<img width="497" alt="Screenshot 2023-09-01 at 12 55 32" src="https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/62558753/d128e2e9-caf0-4aa0-b74d-36f832ca4c28">

Reply:
it cannot work properly.